src/main/java/entity/Player.java

```diff
@@ -0,0 +1,51 @@
+package entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+
+import java.util.concurrent.atomic.AtomicLong;
+
+@Entity
+@Data
+@AllArgsConstructor
+
+public class Player {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private static long idCounter = 1;
+    private long playerID;
```

Comment:
```suggestion
    private long id;
```
nombralo como id, recuerda DRY